src/vs/workbench/browser/parts/editor/editorGroupView.ts

```diff
@@ -945,6 +946,11 @@ export class EditorGroupView extends Themable implements IEditorGroupView {
 		this.element.classList.toggle('active', isActive);
 		this.element.classList.toggle('inactive', !isActive);
 
+		// Unselect editors if group is no longer active
+		if (!isActive && this.selectedEditors.length > 1) {
```

Comment:
Why are we now clearing selections?

Reply:
It's up for debate, but I think it makes sense to clear the selection of an inactive group as the selection would not be drawn anymore and it is still possible to run actions from the context menu of an editor in an inactive group